ref/feature/spnego/examples.adoc

```diff
@@ -3,8 +3,8 @@
 === Default SPNEGO configuration
 
 Adding the SPNEGO feature automatically enforces a certain minimum configuration.
-You don't have to specify a `<spnego>` element in the `server.xml` file.
-Without specifying a `<spnego>` element, the following config is implicit:
+You don't have to specify a `spnego` element in the `server.xml` file.
+Without specifying a `spnego` element, the following config is implicit:
```

Comment:
Without specifying a `spnego` element

suggested edit

If you do not specify a `spnego` element